MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -271,6 +273,10 @@ public class Campaign implements ITechManager {
     private FameAndInfamyController fameAndInfamy;
     private List<Unit> automatedMothballUnits;
 
+     //options relating to parts in use and restock
+    private boolean ignoreMothballed, topUpWeekly;
```

Comment:
Having a twinned variable isn't inherently wrong, but it's not something I've seen elsewhere in our codebase. Strictly personnel preference: I'd define these separately. However, I want to emphasize that this is personal preference so go with whatever your heart tells you. :)